src/main.rs

```diff
@@ -86,16 +86,6 @@ on what should be in that file.");
         None => PathBuf::new(),
     };
 
-    let data_path = match xdg_dirs.create_data_directory(&data_path) {
```

Comment:
You should not remove it here, because it is passed into `conf::load`, but rather you should check inside `conf::load` if the path is empty.